.gitignore.txt

```diff
@@ -0,0 +1,2 @@
+test.md
+node_modules/

```

Comment:
- [ ] As I mentioned in the previous comment message, make sure you correct the typing error of the the `.gitignore` (**.gitignore.text** to **.gitignore**)file and as `the test.md` file is ignored by git, make sure it doesn't appear on GitHub repo.